commit 7dcdfbcf6749cdc4c63e2613cbb3e2392d2fc2fb
Author: Tulio Magno Quites Machado Filho <tuliom@linux.vnet.ibm.com>
Date:   Fri Jun 23 09:10:32 2017 -0300

    powerpc: Update AT_HWCAP[2] bits
    
    Linux commit ID a4700a26107241cc7b9ac8528b2c6714ff99983d reserved 2 more
    bits for the instructions darn (Deliver a Random Number) and scv (System
    Call Vectored).
    
    Linux commit ID 6997e57d693b07289694239e52a10d2f02c3a46f reserved
    another bit for internal usage.
    
            * sysdeps/powerpc/bits/hwcap.h: Add PPC_FEATURE2_DARN and
            PPC_FEATURE2_SCV.
            * sysdeps/powerpc/dl-procinfo.c (_dl_powerpc_cap_flags): Add scv
            and darn.

diff --git a/sysdeps/powerpc/bits/hwcap.h b/sysdeps/powerpc/bits/hwcap.h
index c9daeed..dfc71c2 100644
--- a/sysdeps/powerpc/bits/hwcap.h
+++ b/sysdeps/powerpc/bits/hwcap.h
@@ -50,6 +50,7 @@
 #define PPC_FEATURE_ARCH_2_06	    0x00000100 /* ISA 2.06 */
 #define PPC_FEATURE_HAS_VSX	    0x00000080 /* P7 Vector Extension.  */
 #define PPC_FEATURE_PSERIES_PERFMON_COMPAT  0x00000040
+/* Reserved by the kernel.	    0x00000004  Do not use.  */
 #define PPC_FEATURE_TRUE_LE	    0x00000002
 #define PPC_FEATURE_PPC_LE	    0x00000001
 
@@ -69,3 +70,5 @@
 #define PPC_FEATURE2_ARCH_3_00	   0x00800000 /* ISA 3.0 */
 #define PPC_FEATURE2_HAS_IEEE128   0x00400000 /* VSX IEEE Binary Float
 						 128-bit */
+#define PPC_FEATURE2_DARN	   0x00200000 /* darn instruction.  */
+#define PPC_FEATURE2_SCV	   0x00100000 /* scv syscall.  */
diff --git a/sysdeps/powerpc/dl-procinfo.c b/sysdeps/powerpc/dl-procinfo.c
index cd7329b..4dac16d 100644
--- a/sysdeps/powerpc/dl-procinfo.c
+++ b/sysdeps/powerpc/dl-procinfo.c
@@ -62,7 +62,7 @@ PROCINFO_CLASS const char _dl_powerpc_cap_flags[64][10]
     "", "", "", "",
     "", "", "", "",
     "", "", "", "",
-    "", "", "ieee128", "arch_3_00",
+    "scv", "darn", "ieee128", "arch_3_00",
     "htm-nosc", "vcrypto", "tar", "isel",
     "ebb", "dscr", "htm", "arch_2_07",
   }
